docs/section5/Rule5-3.md

```diff
@@ -24,20 +24,20 @@
 
 - For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
 
-  - For each thermal_block from building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
 
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```  
+  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
 
-  - For each space in thermal zone: ```space_baseline in zone_baseline.spaces:```  
+  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
 
     - Get space conditioning type: ```space_conditioning_type_baseline = space_baseline.conditioning_type```  
 
-      - Get baseline contruction from Table G3.4-1 to G3.4-8 based on space conditioning type, status type and function type: ```surface_performance_target = data_lookup(climate_zone,space_conditioning_type_baseline,"Walls, Below-Grade")```  
+      - Get baseline contruction from Table G3.4-1 to G3.4-8 based on climate zone, space conditioning type and function type: ```surface_performance_target = data_lookup(table_G3_4,climate_zone,space_conditioning_type_baseline,"Walls, Below-Grade")```  
```

Comment:
adding the reference to the table is good, make sure it is done on the other "data_lookups"